packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IForegroundColorTokens, FontTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { View, ViewProps } from 'react-native';
+
+export const tabsName = 'Tabs';
+
+export interface TabsContextData {
+  /*
+   ** The currently selected TabsItem's key
+   */
+  selectedKey: string | null;
+
+  /*
+   ** Index of currently selected key
+   */
+  getTabId?: (key: string, index: number) => string | null;
+
+  /*
+   ** Updates the selected tabsItem and calls the client’s onTabsClick callback
+   */
+  onTabsClick?: (key: string) => void;
+
+  /*
+   ** Updates the selected tabsItem's ref to set as the default tabbable element
+   */
+  updateSelectedTabsItemRef?: (ref: React.RefObject<any>) => void;
+
+  /*
+   ** Array of tabsItem keys in the group
+   */
+  tabsItemKeys?: string[];
+
+  /*
+   ** A Map to for a TabItems corresponding view
+   */
+  views?: Map<string, React.ReactNode[]> | null;
+}
+
+export interface TabsInfo {
+  headersOnly?: boolean;
```

Comment:
We have Info on the state to decide if a slot will be rendered, which is a behavior we found on other components such as Button. I believe that this is because you can't directly access user props on the render.